pkg/controller/control.go

```diff
@@ -310,8 +310,14 @@ func (c *Controller) finishExpansion(expanded bool, size int64) {
 	defer c.Unlock()
 
 	if expanded {
-		logrus.Infof("Controller succeeded to expand from size %v to %v", c.size, size)
+		if c.lastExpansionError != "" {
+			logrus.Infof("Controller succeeded to expand from size %v to %v but there are some replica expansion failures: %v", c.size, size, c.lastExpansionError)
```

Comment:
Should we make the message as below to avoid any confusion about successful expansion?

"Controller finished to expand from ...."

Reply:
Word "finished" is vague... And to avoid confusion, I add one more comment "but..."

Reply:
Ok, I am fine with that.